src/components/TalentsListPage/TalentsList/TalentsList.jsx

```diff
@@ -0,0 +1,5 @@
+import styles from "../TalentsList/TalentsList.module.scss";
+
+export function TalentsList() {
+    return <div className={styles["talent-lists"]}></div>;
```

Comment:
if you wanted to make a div container for the list, the class name should be like talents-list
```suggestion
    return <div className={styles["talents-list"]}></div>;
```